joss-paper/paper.md

```diff
@@ -0,0 +1,202 @@
+---
+title: '21cmFAST v3: A Python-integrated C code for generating 3D realizations of the cosmic 21cm signal.'
+tags:
+  - Python
+  - astronomy
+  - cosmology
+  - simulation
+authors:
+  - name: Steven G. Murray
+    orcid: 0000-0003-3059-3823
+    affiliation: 1
+  - name: Bradley Greig
+    orcid: 0000-0002-4085-2094
+    affiliation: 2, 3
+  - name: Andrei Mesinger
+    orcid: 0000-0003-3374-1772
+    affiliation: 4
+  - name: Julian B. Mu\~noz
+    orcid: 0000-0002-8984-0465
+    affiliation: 5
+Department o
+affiliations:
+ - name: School of Earth and Space Exploration, Arizona State University, Phoenix, USA
+   index: 1
+ - name: ARC Centre of Excellence for All-Sky Astrophysics in 3 Dimensions (ASTRO 3D)
+   index: 2
+ - name: School of Physics, University of Melbourne, Parkville, VIC 3010, Australia
+   index: 3
+ - name: Scuola Normale Superiore, Piazza dei Cavalieri 7, 56126 Pisa, Italy
+   index: 4
+ - name: Department of Physics, Harvard University, 17 Oxford St., Cambridge, MA, 02138, USA
+   index: 5
+date: 29 Feb 2020
+bibliography: paper.bib
+---
+
+# Summary
+
+The field of 21cm cosmology -- in which the hyperfine spectral line of neutral hydrogen
```

Comment:
I would hyphenate this 21-cm, and put a space in the next one. I don't think 21 and cm should go together (other than in the name of the code :)

Reply:
Awesome, yup makes sense.